spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/support/JdbcQueryMethod.java

```diff
@@ -53,13 +59,49 @@ public JdbcQueryMethod(Method method, RepositoryMetadata metadata, ProjectionFac
 	 *
 	 * @return May be {@code null}.
 	 */
+
 	@Nullable
-	String getAnnotatedQuery() {
+	public String getAnnotatedQuery() {
```

Comment:
These methods shouldn't be public. 
Same for `getQueryValue`, `getQueryName` and `getNamedQuery`.